core/src/main/java/org/apache/gravitino/metalake/MetalakeManager.java

```diff
@@ -365,6 +412,7 @@ public void disableMetalake(NameIdentifier ident) throws NoSuchMetalakeException
           try {
             boolean inUse = metalakeInUse(store, ident);
             if (inUse) {
+              METALAKE_CACHE.invalidate(ident);
```

Comment:
cache the result by the way?

Reply:
Why do we need to cache it after performing `disableMetalake` operation?

Reply:
because 416 line update the store

Reply:
done